frontend/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  extends: [
+    'airbnb-base',
+    'plugin:react/recommended',
+    'prettier',
+    'prettier/react',
+  ],
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    'jest/globals': true,
+  },
+  plugins: ['import', 'react', 'jest', 'prettier'],
+  rules: {
+    'prettier/prettier': 'error',
+    'import/no-extraneous-dependencies': ['error', { 'devDependencies': ['**/*.test.js', 'src/setupTests.js'] }],
+    'import/no-unresolved': 0,
+    'import/extensions': 0,
+    'func-names': 0,
+    'no-plusplus': ['error', { allowForLoopAfterthoughts: true }],
```

Comment:
I miss other rules from `cat`
```yml
'no-underscore-dangle': 0, // because of _super
'react/prop-types': 0,
'no-console': ['error', { allow: ['info', 'warn', 'error'] }], // later we will might need wrapper
```
why not adding them?

Reply:
I didn't found agreement about 'no-console' conf; did you?
Are we accepting `console` things?
Why not using a [wrapper as in  `CAT`](https://github.com/src-d/code-annotation/blob/master/src/services/log.js)?